app/src/main/res/layout/list_item_sound.xml

```diff
@@ -9,11 +9,17 @@
             type="com.example.beatboxapp.SoundViewModel" />
     </data>
 
-    <Button
+    <FrameLayout
```

Comment:
Why do we need the FrameLayout?

Reply:
Ok, that makes sense. Indeed, the GridLayout needs to fill in all the space and it is why it would stretch the button horizontally. The framelayout takes up the remaining space and allows the button to center in the available width

Reply:
Can you also check if removing the FrameLayout, the UI breaks? @alexandragnj

Reply:
I did, but I noticed that space between the button and the left edge of the screen is smaller than the space between the button and the right edge of the screen
![Uploading E117EAD8-B39D-4420-91E3-91E8097E38B5.jpeg…]()

Reply:
ok, thanks for checking. Then leave the `FrameLayout` in